python/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -653,3 +658,349 @@ def test_exception_one_label(fit_intercept, client):
     lr = LogisticRegression(fit_intercept=fit_intercept)
     with pytest.raises(ValueError, match=err_msg):
         lr.fit(X, y)
+
+
+@pytest.mark.mg
+@pytest.mark.parametrize("fit_intercept", [False, True])
+@pytest.mark.parametrize(
+    "regularization",
+    [
+        ("none", 1.0, None),
+        ("l2", 2.0, None),
+        ("l1", 2.0, None),
+        ("elasticnet", 2.0, 0.2),
+    ],
+)
+@pytest.mark.parametrize("datatype", [np.float32])
+@pytest.mark.parametrize("delayed", [False])
+@pytest.mark.parametrize("n_classes", [2, 8])
+def test_standardization_on_normal_dataset(
+    fit_intercept, regularization, datatype, delayed, n_classes, client
+):
+
+    penalty = regularization[0]
+    C = regularization[1]
+    l1_ratio = regularization[2]
+
+    # test correctness compared with scikit-learn
+    test_lbfgs(
+        nrows=1e5,
+        ncols=20,
+        n_parts=2,
+        fit_intercept=fit_intercept,
+        datatype=datatype,
+        delayed=delayed,
+        client=client,
+        penalty=penalty,
+        n_classes=n_classes,
+        C=C,
+        l1_ratio=l1_ratio,
+        standardization=True,
+    )
+
+
+@pytest.mark.mg
+@pytest.mark.parametrize("fit_intercept", [False, True])
+@pytest.mark.parametrize(
+    "regularization",
+    [
+        ("none", 1.0, None),
+        ("l2", 2.0, None),
+        ("l1", 2.0, None),
+        ("elasticnet", 2.0, 0.2),
+    ],
+)
+@pytest.mark.parametrize("datatype", [np.float32])
+@pytest.mark.parametrize("delayed", [False])
+@pytest.mark.parametrize("ncol_and_nclasses", [(2, 2), (6, 4)])
```

Comment:
Can we test for a few different variations here please? Even just a couple higher numbers like `(100, 10)` would help.

Reply:
Sure! Just added (100, 10)